missing-number/anniemon.js

```diff
@@ -0,0 +1,19 @@
+/**
+ * 시간 복잡도: nums.length + 1만큼 순회하므로 O(n)
+ * 공간 복잡도: totalSum, numsSum 변수를 사용하므로 O(1)
+ */
+/**
+ * @param {number[]} nums
+ * @return {number}
+ */
+var missingNumber = function(nums) {
+  let totalSum = 0;
+  for(let i = 0; i <= nums.length; i++) {
+      totalSum += i;
+  }
```

Comment:
이렇게 하면 for loop 조회를 줄일 수 있을 것 같아요
```suggestion
  const n = nums.length;
  const totalSum = n * (n + 1) / 2;
```

Reply:
오 좋네요
반영했습니다:)